v1/api/db.py

```diff
@@ -0,0 +1,394 @@
+""" This class sets up the project's database, and commits data to
+    to its respective table
+
+    Call create_tables() or drop_tables() to create or drop the database tables
+
+"""
+
+###
+###### Note : Executing this file directly creates the required tables
+###
+import psycopg2
+from werkzeug.security import check_password_hash
+
+
+class DatabaseManager(object):
+    """ Manages database operations for the API """
+
+    def __init__(self):
+        self.connection_str = "dbname='mydiary' user='postgres' password='postgres'"
+
+    def connect_db(self):
+        """ Returns an active database connection to mydiary database """
+        conn = None
+
+        try:
+            conn = psycopg2.connect(self.connection_str)
+        except psycopg2.DatabaseError as ex:
+            print(ex.pgerror)
+
+        return conn
+
+    @staticmethod
+    def insert_user(fname, lname, uname, email, password):
+        """ Inserts new user into database """
+        result = ''
+        error = ''
+
+        insert_query = """INSERT INTO users(firstname,lastname,username,email,password)
+        VALUES('{0}','{1}','{2}','{3}','{4}') RETURNING uid;"""
+
+        insert_query = insert_query.format(fname, lname, uname, email,
+                                           password)
+
+        dbm = DatabaseManager()
+        conn = dbm.connect_db()
```

Comment:
Reference your connect_db method with self to avoid creating class instances with in methods.